evap/evaluation/models.py

```diff
@@ -1054,6 +1064,15 @@ def __send_to_user(cls, user, template, subject_params, body_params, cc):
         else:
             cc_addresses = []
 
+        send_separate_login_link = False
+        body_params['login_url'] = ""
+        if user.is_external:
```

Comment:
maybe `user.needs_login_key` is more appropriate?

actually, same goes for all the email templates in the test data...